tests/unit_tests/callbacks/tracers/test_tracer.py

```diff
@@ -98,6 +98,7 @@ def test_persist_run(
             lang_chain_tracer_v2.run_map[str(run.id)] = run
         for run in sample_runs:
             lang_chain_tracer_v2._end_trace(run)
+        lang_chain_tracer_v2._executor.shutdown(wait=True)
```

Comment:
would someone ever need to do this in their own code? eg. if running langchain in a serverless environment this could potentially be needed to be really sure the runs are persisted

Reply:
Yeah I think this could end up not submitting tasks if the program exits . I'll look into other options